viite-UI/src/model/ProjectChangeInfoModel.js

```diff
@@ -0,0 +1,29 @@
+(function(root) {
+  root.ProjectChangeInfoModel = function(backend) {
+
+    var roadInfoList=[{endAddressM:1,endRoadPartNumber:0,roadNumber:0,startAddressM:0,startRoadPartNumber:0,trackCode:0}];
```

Comment:
was way to initialize variables for objects so idea would suggest them (to avoid typos)